command-parser.js

```diff
@@ -309,7 +309,7 @@ class CommandContext {
 				connection.popup("Your message can't be blank.");
 				return false;
 			}
-			if (message.length > MAX_MESSAGE_LENGTH && !user.can('ignorelimits')) {
+			if (message.length > MAX_MESSAGE_LENGTH && !user.can('ignorelimits') && message.substr(0, 8) !== '!htmlbox') {
```

Comment:
I opened a local server and it seemed to work. If you want I can do a more thorough test when I get home.